README.md

```diff
@@ -477,7 +491,6 @@ Check out [our other projects][github], [follow us on twitter][twitter], [apply
   [email]: https://cpco.io/email?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=email
   [commercial_support]: https://cpco.io/commercial-support?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=commercial_support
   [we_love_open_source]: https://cpco.io/we-love-open-source?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=we_love_open_source
-  [terraform_modules]: https://cpco.io/terraform-modules?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=terraform_modules
```

Comment:
Deleting this line is failig the linting process. Did you delete it manually? The `README.md` is generated dynamically.

Reply:
Honestly not certain how that happened. Sorry I'm having such a hard time with this one.